fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentLiveTestsTemplate.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * // Copyright (c) Microsoft Corporation. All rights reserved.
+ * // Licensed under the MIT License.
+ */
+
+package com.azure.autorest.fluent.template;
+
+import com.azure.autorest.fluent.model.clientmodel.*;
+import com.azure.autorest.model.javamodel.JavaFile;
+import com.google.common.collect.Lists;
+
+public class FluentLiveTestsTemplate {
+
+    private static final FluentLiveTestsTemplate INSTANCE = new FluentLiveTestsTemplate();
+
+    public static FluentLiveTestsTemplate getInstance(){
+        return INSTANCE;
+    }
+
+    public void write(FluentLiveTests liveTests, JavaFile javaFile) {
+        // write class
+        addImports(liveTests, javaFile);
+        javaFile.publicClass(Lists.newArrayList(), liveTests.getClassName(), classBlock->{
+            for (FluentLiveTestCase testCase : liveTests.getTestCases()) {
+                // write manager field
+                classBlock.privateMemberVariable(liveTests.getManagerType().getName(), liveTests.getManagerName());
+                // write @BeforeEach
+                classBlock.annotation("BeforeEach");
+                classBlock.publicMethod("void init()", methodBlock -> {
+                    methodBlock.line(
+                            String.format("%s = %s.authenticate(" +
+                                    "new DefaultAzureCredentialBuilder().build(), new AzureProfile(AzureEnvironment.AZURE)" +
+                                    ");", liveTests.getManagerName(), liveTests.getManagerType().getName())
+                    );
+                });
+                // write method signature
+                classBlock.annotation("Test");
+                String methodSignature = String.format("%s %s()", "void", testCase.getMethodName());
+                if (testCase.getHelperFeatures().contains(FluentExampleTemplate.HelperFeature.ThrowsIOException)) {
+                    methodSignature += " throws IOException";
+                }
+                classBlock.publicMethod(methodSignature, methodBlock -> {
+                    for (FluentLiveTestStep step : testCase.getSteps()) {
+                        if (step instanceof FluentExampleLiveTestStep) {
+                            FluentExampleLiveTestStep exampleStep = (FluentExampleLiveTestStep) step;
+                            methodBlock.line(exampleStep.getExampleMethod().getMethodContent());
```

Comment:
If FluentLiveTestStep or its source data contains descriptive field like "create signalr resource", consider generating a comment with the description for each step.

Reply:
Sure, just about to do that.